spec/schema.json

```diff
@@ -525,6 +540,225 @@
         "start_date"
       ]
     },
+    "TrustData": {
+      "type": "object",
+      "properties": {
+        "trust_id": {
+          "type": "string"
+        },
+        "trust_name": {
+          "type": "string"
+        },
+        "trust_creation_date_day": {
+          "type": "string"
+        },
+        "trust_creation_date_month": {
+          "type": "string"
+        },
+        "trust_creation_date_year": {
+          "type": "string"
+        },
+        "unable_to_obtain_all_trust_info": {
+          "type": "boolean"
+        },
+        "INDIVIDUAL": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IndividualBeneficialOwner"
+          }
+        },
+        "CORPORATE": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CorporateBeneficialOwner"
+          }
+        },
+        "HISTORICAL_BO": {
+          "type": "string"
```

Comment:
i think HISTORICAL_BO needs a type defined to match the HistoricalBoDto class (maybe TrustHistoricalBoDto if we rename the other 2 types)